src/plopp/functions/scatter3d.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from ..core import input_node
+
+import scipp as sc
+from typing import Union, Literal, Tuple
+
+
+def scatter3d(da: sc.DataArray,
+              *,
+              x: str = None,
+              y: str = None,
+              z: str = None,
+              dim: str = None,
+              figsize: Tuple[Union[int, float]] = None,
+              norm: Literal['linear', 'log'] = 'linear',
+              title: str = None,
+              vmin: sc.Variable = None,
+              vmax: sc.Variable = None,
+              cmap: str = None,
+              **kwargs):
+    """Make a three-dimensional scatter plot.
+
+    To specify the positions of the scatter points, you can use:
+
+    - a single coordinate inside the supplied data array that has dtype ``vector3``
+      (use the ``dim`` parameter to specify the name of the coordinate).
+    - three coordinates from the data array, whose names are specified using the
+      ``x``, ``y``, and ``z`` arguments.
+
+    Note that if ``dim`` is used, ``x``, ``y``, and ``z`` must all be ``None``.
+
+    Parameters
+    ----------
+    da:
+        The data array containing the data and the coordinates.
+    x:
+        The name of the coordinate that is to be used for the X positions.
+    y:
+        The name of the coordinate that is to be used for the Y positions.
+    z:
+        The name of the coordinate that is to be used for the Z positions.
+    dim:
```

Comment:
`dim` is not a good name. Maybe `coord`?

Reply:
Yeah, `dim` is not the best. Maybe `pos`, or `positions`, or `vectors` ?